interface/main/messages/js/reminder_appts.js

```diff
@@ -0,0 +1,524 @@
+           'pid'        : pid,
+           'action'     : 'new_recall'
+           }
+           }).done(function(result) {
+                   obj = JSON.parse(result);
+                   if (obj.DOLV >'') {
+                       //check to see if this is an already scheduled appt for the future
+                       //if so, do you really want a recall? Ask.
+                       //Maybe that appt needs to be removed...
+                        var now = moment(); //new Date()).format('YYYY-MM-DD'); //todays date
+                        var dolv = moment(obj.DOLV); // another date
+                        var duration = dolv.diff(now,'days');
+                        if (duration > '0') { //it's a future appt dude!
+                            alert('NOTE:  This patient has an appointment scheduled for ' + obj.DOLV +' already...');
+                        }
+                   }
+                   $(".news").removeClass('nodisplay');
+                   $("#new_pid").val(obj.pid);
+                   $("#new_phone_home").val(obj.phone_home);
+                   $("#new_phone_cell").val(obj.phone_cell);
+                   if (obj.hipaa_allowsms =="NO") {
+                        $('#new_allowsms_no').prop('checked',true);
+                   } else {
+                        $('#new_allowsms_yes').prop('checked',true);
+                   }
+                   if (obj.hipaa_allowemail =='NO') {
+                        $("#new_email_no").prop('checked',true);
+                   } else {
+                        $("#new_email_yes").prop('checked',true);
+                   }
+                   if (obj.hipaa_voice =='NO') {
+                        $("#new_voice_no").prop('checked',true);
+                   } else {
+                        $("#new_voice_yes").prop('checked',true);
+                   }
+                   $("#new_address").val(obj.street);
+                   $("#new_city").val(obj.city);
+                   $("#new_state").val(obj.state);
+                   $("#new_postal_code").val(obj.postal_code);
+                   $("#new_DOB").html(obj.DOB);
+                   $("#new_email").val(obj.email);
+                   if (obj.DOLV >'') {
+                        $("#DOLV").val(obj.DOLV);
+                   } else {
+                   var today = moment().format('YYYY-MM-DD');
+                   $("#DOLV").val(today);
+                   }
+                   //there is an openemr global for age display under X years old (eg. under "2", so == 17 months old)
+                   //not sure where it is though... or if we can use it here.
+                   $("#new_age").html(obj.age+' years old');
+                   $("#new_reason").val(obj.PLAN);
+                         
+                   });
+}
+
+/**
+ *  This function is called with pressing Submit on the Add a Recall page
+ */
+function add_this_recall(e) {
+    if ($('#datepicker2').val() =='') {
+        alert('Please decide on a Recall Date');
```

Comment:
Use translation strategy.
Here's where you translated another field:
line 67 of interface/main/messages/messages.php
(rec using same strategy and what we usually do here is create a js array when have more than 1 that are then used by the js script)